reporter/pubsub/pubsub.go

```diff
@@ -0,0 +1,109 @@
+package pubsub
```

Comment:
I think a more accurate name for the package and the folder would be `gcpubsub`. `pubsub` alone feels quite generic and misleading. What do you think?

Reply:
fixed